hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsCountersImpl.java

```diff
@@ -332,7 +332,7 @@ public String toString() {
       }
     }
     if (abfsReadFooterMetrics != null) {
-      if (getAbfsReadFooterMetrics().getTotalFiles() > 0 && getAbfsReadFooterMetrics().getTotalReadCount() > 0) {
+      if (getAbfsReadFooterMetrics().getTotalFiles() > 0) {
```

Comment:
Does this guarantee that the file qualified for metric collection ? The read count check was added as many files had 0 reads as per our criteria

Reply:
Previously, we stored the read count to calculate the average. Now that we use mean IOStatistics metrics, storing the read count is no longer necessary..